agot-bg-game-server/src/client/game-state-panel/SelectHouseCardComponent.tsx

```diff
@@ -34,7 +34,7 @@ export default class SelectHouseCardComponent extends Component<GameStateCompone
                                         <Col xs="auto" key={hc.id}>
                                             <HouseCardComponent
                                                 houseCard={hc}
-                                                size="tiny"
+                                                size="small"
```

Comment:
Why this change in particular?

Reply:
@Longwelwind I played around with tiny and small for the chooseHcComp and realized that for selection of a house card small fits better here, especially on small screens. So I changed that here also to have a consistently look and feel when it comes to HC selection.

Reply:
Ok.